### PR TITLE
FIX: numpy.linspace no longer accepts float for num variable

### PR DESCRIPTION
**Pull request by owtscharenko**
As stated in the title, with version 1.18 numpy.linspace no longer accepts float type input variables. The easiest fix for that is a simple cast to int.

With a second commit the simulation example was updated to comply with the function name and variable changes.

### CODE REVIEW

**Comment by owtscharenko** [comment]
added another commit which updates the simulation example.

File: beam_telescope_analysis/examples/simulated_data.py [hunk 1]
```diff
@@ -75,100 +76,105 @@ def run_analysis(n_events):
...
-        hit_analysis.cluster_hits(input_hits_file=data_file,
-                                  min_hit_charge=1,
-                                  max_hit_charge=2 ** 16,
-                                  column_cluster_distance=1,
-                                  row_cluster_distance=1,
-                                  frame_cluster_distance=2,
-                                  dut_name=data_files[i])
+    cluster_files = hit_analysis.cluster(telescope_configuration = initial_configuration,
+                        input_hit_files=data_files,
+                        select_duts = None,
```
Comment (YannickDieter) [inline beam_telescope_analysis/examples/simulated_data.py:97]: PEP8 style here

File: beam_telescope_analysis/examples/simulated_data.py [hunk 2]
```diff
@@ -75,100 +76,105 @@ def run_analysis(n_events):
...
+                        input_mask_files = [None]*sim.n_duts,
+                        use_positions = [False]*sim.n_duts,
+                        min_hit_charges=[1]*sim.n_duts,
+                        max_hit_charges=[2 ** 16]*sim.n_duts,
```
Comment (YannickDieter) [inline beam_telescope_analysis/examples/simulated_data.py:101]: PEP8 style here

**Comment by owtscharenko** [comment]
my bad, I now  ran autopep8 over it.

**Pull request merged**